src/display/api.js

```diff
@@ -783,14 +783,15 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      * @param {number} scale The desired scale of the viewport.
      * @param {number} rotate Degrees to rotate the viewport. If omitted this
      * defaults to the page rotation.
+     * @param {boolean} dontFlip if true, axis Y will not be flipped.
      * @return {PageViewport} Contains 'width' and 'height' properties
      * along with transforms required for rendering.
      */
-    getViewport: function PDFPageProxy_getViewport(scale, rotate) {
-      if (arguments.length < 2) {
-        rotate = this.rotate;
-      }
-      return new PageViewport(this.view, scale, rotate, 0, 0);
+    getViewport: function PDFPageProxy_getViewport(
+        scale,
+        rotate = this.rotate,
+        dontFlip = false) {
```

Comment:
We don't use that kind of formatting anywhere; so as previously suggested, please just replace the *entire* method signature like this (which is valid in ES6):
```javascript
getViewport(scale, rotate = this.rotate, dontFlip = false) {
```

Reply:
Linter will fail on this line as it extends 80 characters. Should I add some comment to make linter ignore this line?

Reply:
> Linter will fail on this line as it extends 80 characters.

No it won't, if you actually change it *exactly* as asked :-)
For the method signature, your diff should contain something like this (not including other relevant changes):
```diff
diff --git a/src/display/api.js b/src/display/api.js
index bd4d404b..1fdeb803 100644
--- a/src/display/api.js
+++ b/src/display/api.js
@@ -783,7 +783,7 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      * @return {PageViewport} Contains 'width' and 'height' properties
      * along with transforms required for rendering.
      */
-    getViewport: function PDFPageProxy_getViewport(scale, rotate) {
+    getViewport(scale, rotate = this.rotate, dontFlip = false) {
       if (arguments.length < 2) {
         rotate = this.rotate;
       }

```

Reply:
ahh, right. I left function name `PDFPageProxy_getViewport` in the first place. Ok, I'll change it.

- [ ] Change function notation